yt/frontends/gadget/data_structures.py

```diff
@@ -50,26 +50,34 @@ def _fix_unit_ordering(unit):
     return unit
 
 
-def _get_gadget_format(filename):
+def _byte_swap_32(x):
```

Comment:
This should always result in a big endian, right?  Is it really swapping, or just converting to big-endian?

Reply:
Yes, in the usage below, it's always converting from little-endian to big-endian. But since the conversion either way is the same operation, I just use this neutral name for the process.

Reply:
Oh, sheesh, you're right.  I got confused in how I was thinking about it.  You're right.  Thank you.